src/util/Util.js

```diff
@@ -113,6 +147,36 @@ define(['okta', 'util/Logger'], function (Okta, Logger) {
     controller.trigger('afterError', { controller: className }, error);
   };
 
+  /**
+   * Why redirect via Form get rather using `window.location.href`?
+   * At the time of writing, Chrome (<72) in Android would block window location change
+   * at following case
+   * 1. An AJAX is trigger because of user action.
+   * 2. 5+ seconds passed without any further user interaction.
+   * 3. User takes an action results in window location change.
+   *
+   * Luckily we discovered that uses Form submit would work around this problem
+   * even though it changed window location.
+   *
+   * Check the commit history for more details.
+   */
+  Util.redirectWithFormGet = function (url) {
+    if (!url) {
+      Logger.error(`Cannot redirect to empty URL: (${url})`);
```

Comment:
Curious, is there an advantage to pass the empty url?

Reply:
not really. it might (not quite likely) be helpful when it shows undefined or null or empty string.